app/views/foods/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Foods</h1>
+
+<section>
+  Number of search results: <%= @foods[:totalHits] %><br>
+  <% @foods[:foods].each do |food| %>
+    <section id="food-<%= food[:fdcId] %>">
+      GTIN/UPC: <%= food[:gtinUpc] %><br>
+      Description: <%= food[:description] %><br>
+      Brand Owner: <%= food[:brandOwner] %><br>
+      Ingredients: <%= food[:ingredients] %><br><br>
+    </section>
+  <% end %>
```

Comment:
Just to reiterate what I mentioned above, `@ foods` is your entire JSON response from your api call. This is unnecessary to send all of this information over, and it can also be dangerous, if there is data in that response that you don't want your user's to have access to.